### DISCUSSION TITLE
rspack support for getCustomTransformers in ts-loader ?

### DISCUSSION THREAD

**Discussion by pushkargupta9807** [web-infra-dev/rspack#3111] [General]
Hello folks, 
is there some example I can refer to if I want to transition from a rule with ts-loader which takes an extra option of "getCustomTransformers" and does some transformations? Is this functionality already available with the inbuilt loaders for rspack?

Thanks!

**Comment by hardfist** [comment]
rspack is not compatible with ts-loader yet

**Comment by toxik** [comment]
hey, can we now use custom transformers?
